apps/meteor/client/views/admin/settings/EditableSettingsProvider.tsx

```diff
@@ -193,9 +193,12 @@ const EditableSettingsProvider = ({ children, query = defaultQuery, omit = defau
 		Tracker.flush();
 	});
 
-	const { data } = useIsEnterprise();
+	const { data } = useLicenseBase({
+		select: (data) => ({ isEnterprise: Boolean(data?.license.license), activeModules: data?.license.activeModules }),
+	});
```

Comment:
I'm not sure. As you can see previously `useIsEnterprise` was called inside the provider in order to provide `isEnterprise` param. What I did was instead of keep the `useIsEnterprise` and add a brand new hook to check the `activeModules`, I called `useLicenseBase` once in order to provide both `isEnterprise` and `activeModules` data

I faced it and I feel curious, but I thought maybe it was for some reason. Do you think we should remove from `EditableSettingProvider`? Any suggestion here?

Reply:
In the long run, `EditableSettingProvider` must be replaced by a RHF form context.

Reply:
I can see the value of the hook `useLicenseActiveModules` I just don't think it needs to be provided by the `EditableSettingsProvider`.  

From what I can see we have two `useIsEnterprise` hooks, one deriving the value from the license and the other from `EditableSettingsProvider` (which in turn uses the first hook anyways 😓 ). 

I think we should remove `isEnterprise` (with the duplicated hook) and `activeModules` from `EditableSettingsProvider`. If this is information is needed it can be accessed directly with [useIsEnterprise](https://github.com/RocketChat/Rocket.Chat/blob/3f9c3f1f5293576b9356b927db6a9d00100e9fac/apps/meteor/client/hooks/useIsEnterprise.ts) or `useLicenseActiveModules` _(if you decide to create the standalone hook)_.